src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -346,6 +405,179 @@ export class PointerTool extends BaseTool {
+
+    // must be set to the opposite side of the mouse if left or bottom
+    const fixDragLeftValue = clickedResizeHandle.includes("left") ? -1 : 1;
+    const fixDragBottomValue = clickedResizeHandle.includes("bottom") ? -1 : 1;
+
+    const staticGlyphControllers = await sceneController.getStaticGlyphControllers();
+
+    await sceneController.editGlyph(async (sendIncrementalChange, glyph) => {
+      const initialPoint = sceneController.selectedGlyphPoint(initialEvent);
+
+      const layerInfo = Object.entries(
+        sceneController.getEditingLayerFromGlyphLayers(glyph.layers)
+      ).map(([layerName, layerGlyph]) => {
+        const behaviorFactory = new EditBehaviorFactory(
+          layerGlyph,
+          sceneController.selection,
+          sceneController.experimentalFeatures.scalingEditBehavior
+        );
+        const layerBounds =
+          staticGlyphControllers[layerName].getSelectionBounds(selection);
+
+        return {
+          layerName,
+          changePath: ["layers", layerName, "glyph"],
+          layerGlyphController: staticGlyphControllers[layerName],
+          editBehavior: behaviorFactory.getBehavior("default", true),
+          regularPinPoint: getPinPoint(layerBounds, origin.x, origin.y),
+          altPinPoint: getPinPoint(layerBounds, undefined, undefined),
+          selectionWidth: layerBounds.xMax - layerBounds.xMin,
+          selectionHeight: layerBounds.yMax - layerBounds.yMin,
+        };
+      });
+
+      let editChange;
+      for await (const event of eventStream) {
+        const currentPoint = sceneController.selectedGlyphPoint(event);
+        const delta = {
+          x: (currentPoint.x - initialPoint.x) * fixDragLeftValue,
+          y: (currentPoint.y - initialPoint.y) * fixDragBottomValue,
+        };
+
+        const deepEditChanges = [];
+        for (const layer of layerInfo) {
+          const layerGlyph = layer.layerGlyphController.instance;
+
+          // NOTE: calculate the scale based on selection width per layer.
+          let scaleX = (layer.selectionWidth + delta.x) / layer.selectionWidth;
+          let scaleY = (layer.selectionHeight + delta.y) / layer.selectionHeight;
+
+          if (clickedResizeHandle.includes("middle")) {
+            if (event.shiftKey) {
+              scaleY = scaleX;
+            } else {
+              scaleY = 1;
+            }
+          }
+          if (clickedResizeHandle.includes("center")) {
+            if (event.shiftKey) {
+              scaleX = scaleY;
+            } else {
+              scaleX = 1;
+            }
+          }
+
+          // scale proportionally if shift key is pressed
+          if (event.shiftKey) {
+            if (scaleX > scaleY) {
+              scaleY = scaleX;
+            } else {
+              scaleX = scaleY;
+            }
+          }
+          let transformation = new Transform().scale(scaleX, scaleY);
+
+          let pinPoint = layer.regularPinPoint;
```

Comment:
Please write this as an `?` expression instead of overwriting a redundant assignment. Like this:
```js
   const pinPoint = event.altKey ? layer.altPinPoint : layer.regularPinPoint;
```
This style is almost always to be preferred.

Reply:
Thanks. done.